apps/storybook-react-native/tailwind-intellisense.config.js

```diff
@@ -7,4 +7,17 @@ module.exports = {
     '../../packages/design-system-react-native/src/**/*.{js,jsx,ts,tsx}',
     './stories/**/*.{js,jsx,ts,tsx}',
   ],
+  theme: {
+    // Keep essential semantic colors, remove default palette colors. We want to rely on the colors provided by the design system preset
+    colors: {
+      inherit: 'inherit',
+      current: 'currentColor',
+      transparent: 'transparent',
+      black: '#000000',
+      white: '#ffffff',
+    },
+    fontSize: {}, // This removes all default Tailwind font sizes. We want to rely on the design system font sizes and enforce use of the Text component
+    extend: {},
+  },
+  plugins: [],
 };
```

Comment:
Updating the tailwind.config.js for intellisense. Going to look at generating this from `tailwind.config.ts` and pointing to the version that we can include in the twrnc preset